venues/ICML.cc/2017/MLAV/python/superuser-init.py

```diff
@@ -0,0 +1,210 @@
+
+	if manual_activation:
+		valid_token = False
+		while not valid_token:
+			try:
+				## PAM what is the confirmation token???
+				token = raw_input("Please provide the confirmation token: ")
+				activation = client.activate_user(token = token)
+				print "Admin account activated."
+				print "UserID: ", activation['user']['profile']['id']
+				print "Login: ", username
+				valid_token = True
+			except:
+				print "Invalid token."
+	else:
+		print "Admin account not activated. Please respond to the email confirmation sent to %s" % username
+	client.add_members_to_group(admin_group, [username])
+	print "Added %s to %s: " % (username, config.CONF)
+
+
+## Submission invitation
+reply = {
+	'forum': None,
+	'replyto': None,
+	'readers': {
+		'description': 'The users who will be allowed to read the above content.',
+		'values': ['everyone']
+	},
+	'signatures': {
+		'description': 'How your identity will be displayed with the above content.',
+		'values-regex': '~.*'
+	},
+	'writers': {
+		'description': 'How your identity will be displayed with the above content.',
+		'values-copied': ['{content.authorids}', '{signatures}']
+	},
+	'content': {
+		'title': {
+			'description': 'Title of paper.',
+			'order': 1,
+			'value-regex': '.{1,250}',
+			'required': True
+		},
+		# [^;,\\n] means all chars except ';' ',' and '\n'
+		'authors': {
+			'description': 'Comma separated list of author names, as they appear in the paper.',
+			'order': 2,
+			'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+			'required': True
+		},
+		'authorids': {
+			'description': 'Comma separated list of author email addresses, in the same order as above.',
+			'order': 3,
+			# a list of email addresses
+			'values-regex': "([a-z0-9_\-\.]{2,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,},){0,}([a-z0-9_\-\.]{2,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,})",
+			'required': True
+		},
+		'abstract': {
+			'description': 'Abstract of paper.',
+			'order': 4,
+			# is this up to 5,000 words, because \\S is not white space followed by \\s which is white space
+			'value-regex': '[\\S\\s]{1,5000}',
+			'required': True
+		},
+		'pdf': {
+			'description': 'Upload a PDF file that ends with .pdf)',
+			'order': 8,
+			'value-regex': 'upload',
+			# 'value-regex': 'upload|http://arxiv.org/pdf/.+',
+			# 'description': 'Either upload a PDF file or provide a direct link to your PDF on ArXiv (direct links must begin with http(s) and end with .pdf)'
+			'required': True
+		},
+		'conflicts': {
+			'description': 'Comma separated list of email domains of people who would have a conflict of interest in reviewing this paper, (e.g., cs.umass.edu;google.com, etc.).',
+			'order': 9,
+			'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+			'required': True
+		},
+	}
+}
+
+utils.process_to_file(
+	utils.get_path("../process/submissionProcess.template", __file__),
+	utils.get_path("../process", __file__),
+)
+
+submit_invitation = openreview.Invitation(config.CONF + '/-/Submission',
```

Comment:
the same here for the invitation. The superuser-init.py should create the main group of the conference and the admin user, that's all, right? @mspector